### PR TITLE
Adding Logoutfeature

### LINKED ISSUE #93: Adding Log out  Feature in open event app
Log out Feature needs to be added in open event app

Would You Like to Work on this issue?
Yes

### PR DESCRIPTION
**Pull request by SIMRAN1**
Fixes #93 
Logout Feature is added in app .
image

### CODE REVIEW

**Comment by SIMRAN1** [comment]
@imjog @chashmeetsingh please review!

File: FOSSAsia/Constants.swift [hunk 1]
```diff
@@ -43,6 +43,13 @@ struct Constants {
 
     }
 
+    struct alertMessage {
+        static let logoutMessageTitle = "Do you want to Logout?"
```
Comment (jogendra) [inline FOSSAsia/Constants.swift:47]: change title message to `Confirm Logout`

File: FOSSAsia/Constants.swift [hunk 2]
```diff
@@ -43,6 +43,13 @@ struct Constants {
...
+        static let logoutMessage = "Click Yes to Logout!"
```
Comment (jogendra) [inline FOSSAsia/Constants.swift:48]: change this to `Are you sure you want to logout?`

File: FOSSAsia/Constants.swift [hunk 3]
```diff
@@ -43,6 +43,13 @@ struct Constants {
...
+        static let noTitle = "No"
```
Comment (jogendra) [inline FOSSAsia/Constants.swift:49]: change this to `Cancel`

File: FOSSAsia/Constants.swift [hunk 4]
```diff
@@ -43,6 +43,13 @@ struct Constants {
...
+        static let yesTitle = "Yes"
```
Comment (jogendra) [inline FOSSAsia/Constants.swift:50]: Change this to `Logout`

**Review by jogendra — APPROVED** [review]
Change logout alert messages. Otherwise looks fine to me.

**Pull request merged**